pkg/stenciltest/stenciltest.go

```diff
@@ -75,12 +79,16 @@ func New(t *testing.T, templatePath string, additionalTemplates ...string) *Temp
 		t.Fatal(err)
 	}
 
+	log := logrus.New()
+	log.SetLevel(logrus.DebugLevel)
```

Comment:
Figured it'd be useful to alway run tests w/ the debug logger to make it easier for people to identify issues